scripts/compare_benchmarks.py

```diff
@@ -123,6 +126,32 @@ def calculate_and_format_p_value(old, new):
 table = AsciiTable(table_data)
 table.justify_columns[6] = 'right'
 
+# If github_format is set, format the output in the style of a diff file where added lines (starting with +) are
+# colored green, removed lines (starting with -) are red, and others (starting with an empty space) are black.
+# Because terminaltables (unsurprisingly) does not support this hack, we need to post-process the result string,
+# searching for the control codes that define lines formatted as green or red.
+
+result = str(table.table)
+if github_format:
+    new_result = ''
+    green_control_sequence = colored('', 'green')[0:5]
+    red_control_sequence = colored('', 'red')[0:5]
+
+    for line in result.splitlines():
+        if green_control_sequence in line:
+            new_result += '+'
+        elif red_control_sequence in line:
+            new_result += '-'
+        else:
+            new_result += ' '
+
+        new_result += line + '\n'
+    result = new_result
+
 print("")
-print(table.table)
+if github_format:
```

Comment:
Why not add those `diff` markers to `new_result` in the conditional above and do a simple `print(result)` here?